plugins/services/src/js/components/forms/MultiContainerNetworkingFormSection.js

```diff
@@ -0,0 +1,451 @@
+import React from 'react';
+import {Tooltip} from 'reactjs-components';
+import mixin from 'reactjs-mixin';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import FieldHelp from '../../../../../../src/js/components/form/FieldHelp';
+import FieldInput from '../../../../../../src/js/components/form/FieldInput';
+import FieldLabel from '../../../../../../src/js/components/form/FieldLabel';
+import FieldError from '../../../../../../src/js/components/form/FieldError';
+import FieldSelect from '../../../../../../src/js/components/form/FieldSelect';
+import {findNestedPropertyInObject} from '../../../../../../src/js/utils/Util';
+import FormGroup from '../../../../../../src/js/components/form/FormGroup';
+import FormGroupContainer from '../../../../../../src/js/components/form/FormGroupContainer';
+import {FormReducer as network} from '../../reducers/serviceForm/MultiContainerNetwork';
+import HostUtil from '../../utils/HostUtil';
+import Icon from '../../../../../../src/js/components/Icon';
+import Networking from '../../../../../../src/js/constants/Networking';
+import VirtualNetworksStore from '../../../../../../src/js/stores/VirtualNetworksStore';
+
+const METHODS_TO_BIND = [
+  'onVirtualNetworksStoreSuccess'
+];
+
+class MultiContainerHealthChecksFormSection extends mixin(StoreMixin) {
```

Comment:
There is a lot of this section that is just a copy of `NetworkingFormSection`. I thought we aimed to build section parts that we could re-use, so we would not end up in this situation. E.g fixing here, without knowing that the same issue is in `NetworkingFormSection`, which then will be forgotten.